### ISSUE TITLE
Uncaught Error: EPERM, operation not permitted 'c:\Program Files (x86)\LTC\LTspiceIV\lib\sym\voltage.asy'

### ISSUE THREAD

**Issue by perlsmith** [atom/atom#5137]
[Enter steps to reproduce below:]

1. Open with Atom on voltage.asy
2. Don't know how to attach the file here..

**Atom Version**: 0.174.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: EPERM, operation not permitted 'c:\Program Files (x86)\LTC\LTspiceIV\lib\sym\voltage.asy'

```
At c:\Users\Jul\AppData\Local\atom\app-0.174.0\resources\app\src\workspace.js:504

Error: EPERM, operation not permitted 'c:\Program Files (x86)\LTC\LTspiceIV\lib\sym\voltage.asy'
  at Error (native)
  at Object.fs.openSync (fs.js:455:18)
  at Object.module.(anonymous function) [as openSync] (c:\Users\Jul\AppData\Local\atom\app-0.174.0\resources\atom\common\lib\asar.js:422:20)
  at Project.module.exports.Project.open (c:\Users\Jul\AppData\Local\atom\app-0.174.0\resources\app\src\project.js:287:31)
  at Workspace.module.exports.Workspace.openURIInPane (c:\Users\Jul\AppData\Local\atom\app-0.174.0\resources\app\src\workspace.js:492:31)
  at Workspace.module.exports.Workspace.open (c:\Users\Jul\AppData\Local\atom\app-0.174.0\resources\app\src\workspace.js:418:19)
  at Ipc.<anonymous> (c:\Users\Jul\AppData\Local\atom\app-0.174.0\resources\app\src\window-event-handler.js:45:32)
  at Ipc.emit (events.js:110:17)
  at process.<anonymous> (c:\Users\Jul\AppData\Local\atom\app-0.174.0\resources\atom\renderer\api\lib\ipc.js:22:29)
  at process.emit (events.js:118:17)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "editor": {
    "fontSize": 18,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by 50Wliu** [comment]
This will be fixed in the next Atom version, 0.175.0.

**Comment by kevinsawicki** [comment]
Fixed by https://github.com/atom/atom/pull/5114
